README.md

```diff
@@ -100,6 +100,18 @@ Instead you must escape **every** special character (notice the `%` escape too)
 
 If `-ListPath` is set to **GPO** the Black/White List can be managed from within the GPO itself under **Application GPO Blacklist**/**Application GPO Whitelist**. Thanks to [Weatherlights](https://github.com/Weatherlights) in [#256 (reply in thread)](https://github.com/Romanitho/Winget-AutoUpdate/discussions/256#discussioncomment-4710599)!
 
+### Use winget to install WAU
+The following command will install WAU through winget itself in the newest version available. 
+
+```
+winget install Romanitho.Winget-AutoUpdate
+```
+
+You can use [Winget-Install](https://github.com/Romanitho/Winget-Install) to deploy the package for example in Intune: 
```

Comment:
The thing is, I will not maintain that repo anymore, as winget-install is part of WAU now.

Reply:
Am I wrong, but right now, not everything has been moved from winget-install to WAU, right? 
"(Will be part of WAU - Work in progress)".

Therefore doesnt it make sense to start like this and then change the README again once everything has merged to WAU?

But of course, I can remove that part if you want to.

Reply:
The script at [Winget-Install.ps1](https://github.com/Romanitho/Winget-AutoUpdate/blob/main/Sources/Winget-AutoUpdate/Winget-Install.ps1) is part of WAU and is maintained here.

The main goal of this script is to install WAU and use it as a dependency for all your Winget packages. This way, you can create empty packages with the following command: "%systemroot%\sysnative\WindowsPowerShell\v1.0\powershell.exe" -ExecutionPolicy Bypass -File "C:\Program Files\Winget-AutoUpdate\Winget-Install.ps1" -AppIDs Adobe.Acrobat.Reader.64-bit

This way, Mods and stuff are also applied.

But i'm open to have a standalone repo with that. But double repo to maintain 😅

Reply:
Thanks understood. Maybe you should then archive your old repo and make a link to WAU and the script there. For example I am using winget-install and WAU since your early days and I always thought that this is still WIP, havent noticed that Winget-install is already completly obsolete. 

For me personally, those are two completely different things (WAU vs. winget-install) as we dont install apps with WAU, WAU is just for updating? But in the end it is how you want to handle your tools :) Have a look at the latest commit, I hope thats fine. 

Until now my package at Microsoft hasnt been approved yet.

Reply:
WAU is now available through Winget. 😅
Thanks.